### ISSUE TITLE
Ski and Goddess Icons on Events Menu Show Up as Squares

### ISSUE THREAD

**Issue by AnaHotaling** [realbig/RBM_TH_EPIC#26]
Ack! 

And could you please tell me how to reopen an issue? Thanks!

**Comment by d4mation** [comment]
Have you tried Hard Refreshing your browser?

CTRL+F5 on a Windows computer, or CMD+SHIFT+R on a Mac.

Also, when you Comment on an Issue, you can click "Reopen and Comment" to re-open Issues.

**Comment by AnaHotaling** [comment]
Ummm... I do know enough to refresh a browser! Apple Help Desk and AAPS IT department background, remember? :)

I've refreshed and I cleared my history, too. Still get the little squares. 

I have never seen the Reopen and Comment box. Let me go try on a closed issue.

**Comment by AnaHotaling** [comment]
Nope, reopening didn't work. Here's a screenshot. 
![screenshot 2016-01-08 13 48 56](https://cloud.githubusercontent.com/assets/16377404/12206397/aab342cc-b60e-11e5-92c2-32e5af87fc13.png)

**Comment by d4mation** [comment]
Might be due to permissions on GitHub then. I'm a Collaborator on the Repository, so I may have have options that you dont.

![image](https://cloud.githubusercontent.com/assets/7770631/12206451/054f9794-b60f-11e5-8f63-6ac6b6a64f9f.png)

What Browser/Operating System are you using? 

I've tested it on Windows 8.1 Pro and Windows 10 Pro in Chrome, Firefox, and IE 11 and I can see the Icons in all of them.

![image](https://cloud.githubusercontent.com/assets/7770631/12206489/40567e3e-b60f-11e5-9494-3a5617d3ebff.png)

Didn't know about your previous IT experience, sorry about that. I'm fairly new around here. Figured I'd cover my bases with basic troubleshooting.

**Comment by AnaHotaling** [comment]
LOL! And you can blame me, too... I just assume Kyle is the one always
responding. Of course you didn't know!

And nuts about reopening issues. Guess we'll just stick with creating new
ones. What a pain. I don't foresee a need to become a collaborator... it's
not like I've been writing any valuable code recently that I feel is worth
sharing! :D

Ana Hotaling
Director of Inbound Marketing, Epic Races

**Comment by AnaHotaling** [comment]
Well, look at that. I CAN reply by email and it shows up here! Whaddya know! That was an accident, but a fortuitious one. Anyway, I'm going to "borrow" my husband's desktop computer and check how the Events menu looks. I just upgraded my laptop to El Capitan OS and I've lost a LOT of functionality that is bugging me big time.

**Comment by AnaHotaling** [comment]
Just tried it on Firefox 43.0.4 (he and I both upgraded to it yesterday). This is what I'm seeing:

![screenshot 2016-01-08 14 02 14](https://cloud.githubusercontent.com/assets/16377404/12206780/cdf0a8ae-b610-11e5-9b9d-8d3c7a7a873e.png)

**Comment by AnaHotaling** [comment]
People still use Exploder????

**Comment by AnaHotaling** [comment]
Let me try Safari. Nope, Safari is fine.

**Comment by d4mation** [comment]
IE? Yes, sadly.

I just booted up on my wife's Macbook Pro running El Capitan and the new icons load just fine in Safari, Chrome, and Firefox on it as well.

I just cleared all the Server Caches I have access to and reset all the File Permissions. I cleared them earlier as well, but perhaps it could be better for you now.

**Comment by AnaHotaling** [comment]
Let me try again, hold on! If not, I'm rebooting all the computers in our office. We're the kind that leave them on 24/7...

**Comment by AnaHotaling** [comment]
Dang it all. I'm rebooting. My son's laptop has it just fine.

**Comment by AnaHotaling** [comment]
Technology is NOT my friend!!!! The reboot didn't do it, but I cleared the cache on my laptop and on our household server, just because I figured it wouldn't hurt. All is good. Sigh. 

Onto the next task! THANKS!

**Comment by d4mation** [comment]
No problem!
